cyclonedds/tools/cli/sub.py

```diff
@@ -3,54 +3,89 @@
 from rich import print
 from rich.syntax import Syntax
 from rich.prompt import Prompt
+from rich.console import Console
 
-from .utils import TimeDeltaParamType, LiveData, background_progress_viewer, background_printer
+from .utils import (
+    TimeDeltaParamType,
+    LiveData,
+    background_progress_viewer,
+    background_printer,
+)
 from .discovery.main import type_discovery
 from .data import subscribe as data_subscribe
 
 
 @click.command(short_help="Subscribe to an arbitrary topic")
 @click.argument("topic")
-@click.option('-i', '--id', '--domain-id', type=int, default=0, help="DDS Domain to inspect.")
-@click.option('-r', '--runtime', type=TimeDeltaParamType(), default='1s', help="Duration of discovery scan.")
-def subscribe(topic, id, runtime):
+@click.option(
+    "-i", "--id", "--domain-id", type=int, default=0, help="DDS Domain to inspect."
+)
+@click.option(
+    "-r",
+    "--runtime",
+    type=TimeDeltaParamType(),
+    default="1s",
+    help="Duration of discovery scan.",
+)
+@click.option(
+    "--suppress-progress-bar",
+    type=bool,
+    is_flag=True,
+    help="Suppress the output of the progress bar",
+)
+@click.option(
+    "--force-color-mode",

```

Comment:
I was reflecting on this option somewhat. I feel as if it would be more appropriate for this to be a `--color` flag which takes an enumeration of `auto`, `always`, or `never`. The behavior of `--force-color-mode` would  then be achieved by  `--color always`.